cspell.json

```diff
@@ -2,158 +2,164 @@
   "version": "0.2",
   "language": "en",
   "words": [
+    "aesthetician",
     "Altius",
-    "BOC-3",
     "Balloonport",
     "Balloonsport",
+    "BOC-3",
+    "brownfields",
     "Brownfields",
+    "businessnjgovnavigator",
```

Comment:
So my IDEA auto alphabetized this. or maybe prettier did. Anyways I just added a few things here like " and "mynj"businessnjgovnavigator"